app/preparations_for_batle/apply_equipment.py

```diff
@@ -3,8 +3,7 @@ def apply_weapon(self) -> None:
         self.power += self.weapon["power"]
 
     def apply_armour(self) -> None:
-        for armour in self.armour:
-            self.protection += armour["protection"]
+        self.protection = sum([armour["protection"] for armour in self.armour])
```

Comment:
sum works with generators so [] are not needed

Reply:
Fixed